misc/requirements/requirements-dev.txt-raw

```diff
@@ -9,6 +9,7 @@ twine
 
 # Included to override setuptools' vendored version that is being included in
 # the lock file by pip freeze.
+importlib_resources
```

Comment:
Needed here now because it's not included transitively via the main requirements anymore, but the problem you describe is still present.